packages/codemod/src/utils/prop-utils.ts

```diff
@@ -173,6 +173,25 @@ export function updatePropValues(
   });
 }
 
+export function updateStaticPropKeys(
+  j: JSCodeshift,
+  elementPath: ASTPath<JSXElement>,
+  propName: string,
+  keysMapping: Record<string, string>
```

Comment:
there's no real need to make it a `Record`, am I right?
since it gets a single propName, so there would always be one key and one value
So you can just get the old static key and new static key or something similar